docs/user_guide/mitigation.rst

```diff
@@ -81,6 +87,560 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+Thresholder
+~~~~~~~~~~~~
+
+What it does
+^^^^^^^^^^^^
+
+:class:`fairlearn.postprocessing.Thresholder` is a post processing technique
+which outputs a binary prediction based on group-specific thresholds. The
+specified thresholds can work in two directions (positive/negative), and can
+be applied to the output of the :code:`predict()` function a regressor,
+:code:`predict_proba()[:,1]` of a classifier, or on
```

Comment:
That `[:,1]` only applies to a binary classifier, right?

Reply:
True, I wrote it like this because if the `predict_method` specified by the user is `predict_proba`, the proba that is used internally is the one obtained by calling `predict_proba()[:,1]` (by calling `_get_soft_predictions()`).